### ISSUE TITLE
As a data engineer, I need to test the new schema, including models for People, Company, Place

### ISSUE THREAD

**Issue by mwigham** [beeldengeluid/beng-lod-server#36]
We have adapted the schema to improve modelling of people, companies and places (#34), and will update the data (#33). Once that is complete, we need to test if the new modelling does what we expect.

- [x] all types of person and companies are modelled and are present in the data
- [x] all types of location are modelled and are present in the data
- [x] the data can be queried with the same pattern of query (note, this is not yet that all people can be queried with the same query, but that the query should be able to get to the GTAA concepts representing the people via the same query structure, not that one type of person relationship requires an additional triple to reach the person)

**Comment by jblom** [comment]
@mwigham @wmelder nice, I'm really looking forward to hearing some more about what you've done. Please consider a small workshop/demonstration where you can show what you've accomplished and how you've went about your work and why. 

I think this could be a nice intermediate result before we start hooking up to DAAN.

Added this to the TODO in sprint

**Comment by mwigham** [comment]
Persons/Organisations:
originalCreators - not in original data
contractors - not in original data
sponsors - not in original data
All other entityInRole types are present. They all point to a subtype of EntityInRole, which points to either a Person or Organisation, which is a concept with a skos:prefLabel, as it should be.
(Note, the only Funder presently in the data has the label "Schone inlassen aanwezig van onderwerp 1,3 en 4." - this is not a fault in the LD, this is the actual archive value...)

Locations:
hasProductionCountry and hasLocation are both present, and both point to a concept with a skos:prefLabel, as they should
hasRecordingLocation is present and is just a string, this is not a thesaurus field in the archive. This means it can't be searched with the same query as the other locations, but that is probably for the best as the values are very different.

Other thesaurus metadata fields:
hasMusicStyle - not in original data
hasLanguage - should be hasLanguageUsage, and should point to a use (which is a label) and a Language, which is a SKOS concept (or do we not want to do this as it is a non GTAA schema?)
network - should be hasBroadcastStation, and should have a skos:Concept as object (or do we not want to do this as it is a non GTAA schema?)
broadcaster - should be hasBroadcaster, and should have a skos:Concept as object (or do we not want to do this as it is a non GTAA schema?)

All other thesaurus metadata fields are present and point to a concept with a skos:prefLabel, as they should.

In summary: some minor niggles which can be addressed later, but the key functionality is perfect.

**Comment by mwigham** [comment]
@jblom I think this can be closed
